coriolis/osmorphing/osmount/windows.py

```diff
@@ -214,16 +215,18 @@ def _set_volumes_drive_letter(self):
                     self._conn.exec_ps_command(
                         f'Set-Partition -NoDefaultDriveLetter $False '
                         f'-DiskNumber {disk_num} -PartitionNumber {part_num}')
+                    disk_nums.append(disk_num)
                 except exception.CoriolisException:
                     LOG.warning(
                         f"Failed setting default drive letter on partition "
                         f"number '{part_num}' of disk number '{disk_num}'. "
                         f"Error was: {utils.get_exception_details()}")
+            self._rebring_disks_online(disk_nums=disk_nums)
 
     def mount_os(self):
+        self._bring_disks_online()
```

Comment:
This should come after `_set_basic_disks_rw_mode` please, but before `_set_volumes_drive_letter`.